config/test-files/systemconfiguration-no-computes.yaml

```diff
@@ -0,0 +1,29 @@
+# Describes the Rabbit nodes created for the KIND environment, and some
+# imaginary compute nodes for those rabbits.
+apiVersion: dataworkflowservices.github.io/v1alpha2
+kind: SystemConfiguration
+metadata:
+  # Nnf-sos expects this resource to be default/default.
+  name: default
+  namespace: default
+spec:
+  computeNodes:
+  - name: compute-01
```

Comment:
Why not empty out the computes and computesAccess lists here?

Reply:
done